modules/@apostrophecms/permission/ui/apos/components/AposInputRole.vue

```diff
@@ -126,7 +126,6 @@ export default {
       const list = document.createElement('ul');
       const intro = document.createElement('p');
       const followUp = document.createElement('p');
-      const link = document.createElement('a');
```

Comment:
not used, throwing an eslint error in the IDE, though our eslint checker clearly doesn't check Vue files.